src/components/UserRepoDate.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import './UserStarsRepo.scss';
+
+function UserRepoDate() {
+  const [repolist, setRepolist] = useState([]);
+
+  useEffect(() => {
+    axios.get('https://api.github.com/users/erriep/repos').then(({ data }) => {
+      setRepolist(data);
+    });
+  }, []);
+
+  return (
+    <div>
+      <p>Liste des 5 repos les plus récents(nom-date-de création-langage)</p>
+      <section className="UserStarsRepo">
+        <div className="cardeList">
+          {repolist
+            .sort(function compare(a, b) {
```

Comment:
You can drop the "compare" function name, it can be a nameless function

Reply:
"compare" deleted